docs/website/content/docs/bootstrapping.md

```diff
@@ -1,28 +1,129 @@
 sidebar: "docs"
 prev: "/docs/installation/"
 next: "/docs/patching/"
 ---
 
 # Bootstrapping
 
+## Introduction
+
+Imagine a scenario in which you have shown up to a datacenter with only a laptop and your task is to transition a rack of bare metal machines into an HA management plane and multiple Kubernetes clusters created by that management plane.
+In this guide, we will go through how to create a bootstrap cluster using a Docker-based Talos cluster, provision the management plane, and pivot over to it.
+Guides around post-pivoting setup and subsequent cluster creation should also be found in the "Guides" section of the sidebar.
+
+Because of the design of Cluster API, there is inherently a "chicken and egg" problem with needing a Kubernetes cluster in order to provision the management plane.
+Talos Systems and the Cluster API community have created tools to help make this transition easier.
+
+## Prerequisites
+
+There are two big prerequisites involved with bootstrapping Sidero: routing and DHCP setup.
+
+From the routing side, the laptop from which you are bootstrapping *must* be accessible by the bare metal machines that we will be booting.
+In the datacenter scenario described above, the easiest way to achieve this is probably to hook the lapop onto the server rack's subnet by plugging it into the top-of-rack switch.
+This is needed for TFTP, PXE booting, and the ability of the metal machines to register themselves with the bootstrap plane.
+
+DHCP configuration is needed to tell the metal servers what their "next server" is when PXE booting.
+The configuration of this is different for each environment and each DHCP server, thus it's impossible to give an easy guide.
+However, here is an example of the configuration from my EdgeRouter that uses vyatta-dhcpd as the DHCP service:
+
+This block shows the subnet setup, as well as the extra "subnet-parameters" that tell the DHCP server to include the ipxe-metal.conf file.
+
+```bash
+$ show service dhcp-server shared-network-name MetalDHCP
+
+ authoritative enable
+ subnet 192.168.254.0/24 {
+     default-router 192.168.254.1
+     dns-server 192.168.1.200
+     lease 86400
+     start 192.168.254.2 {
+         stop 192.168.254.252
+     }
+     subnet-parameters "include &quot;/etc/dhcp/ipxe-metal.conf&quot;;"
+ }
+```
+
+And this is my ipxe-metal.conf file.
+Notice that it sets a static address for the management node that I'll be booting, in addition to providing the "next server" info.
+This "next server" IP address will match references to `PUBLIC_IP` found below in this guide.
+
+```bash
+$ cat /etc/dhcp/ipxe-metal.conf
+
+allow bootp;
+allow booting;
+
+group "management-plane" {
+  next-server 192.168.1.150;
+  if exists user-class and option user-class = "iPXE" {
+    filename "http://192.168.1.150:8081/boot.ipxe";
+  } else {
+    filename "ipxe.efi";
+  }
+
+  host talos-mgmt-0 {
+     fixed-address 192.168.254.2;
+     hardware ethernet d0:50:99:d3:33:60;
+  }
+}
+```
```

Comment:
```suggestion
Here is the ipxe-metal.conf file:

```bash
$ cat /etc/dhcp/ipxe-metal.conf

allow bootp;
allow booting;

group "management-plane" {
  next-server 192.168.1.150;
  if exists user-class and option user-class = "iPXE" {
    filename "http://192.168.1.150:8081/boot.ipxe";
  } else {
    filename "ipxe.efi";
  }

  host talos-mgmt-0 {
     fixed-address 192.168.254.2;
     hardware ethernet d0:50:99:d3:33:60;
  }
}
```

Notice that it sets a static address for the management node that we'll be booting, in addition to providing the "next server" option.
This "next server" IP address will match references to `PUBLIC_IP` found below in this guide.
```

Reply:
GitHub is messing up the format, but I think you get the picture. Just minor reordering.